templates/partials/_code-snippet.html

```diff
@@ -1,4 +1,3 @@
-<div class="p-code-copyable">
-  <input class="p-code-copyable__input" id="snippet-{{ snippet_id }}" value="{{ snippet_value }}" readonly="readonly">
-  <button class="p-code-copyable__action js-clipboard-copy" data-clipboard-target="#snippet-{{ snippet_id }}">Copy to clipboard</button>
+<div class="p-code-snippet">
+  <pre class="p-code-snippet__block--icon"><code id="snippet-{{ snippet_id }}">{{ snippet_value }}</code></pre>
```

Comment:
Do you need the id on `code`? Quite possibly it was only used for copy functionality (which was removed).